umap/views.py

```diff
@@ -1012,7 +1011,7 @@ def render_to_response(self, context, **response_kwargs):
             with open(path, "rb") as f:
                 # Should not be used in production!
                 response = HttpResponse(f.read(), content_type="application/geo+json")
-            response["Last-Modified"] = self.last_modified
+            response["X-Umap-Version"] = self.version
```

Comment:
Why is that UPPERCASED in JS and CamelCased in Python? Is that a convention I miss?

Reply:
Thanks for pointing it out. I started with uppercase everywhere, and then went to camel case, but didn't update it everywhere.